python/sglang/srt/lora/lora_config.py

```diff
@@ -24,24 +24,47 @@ def __init__(
         path: str,
     ) -> None:
         self.path = path
+        self.weights_dir = self._get_weights_dir()
         self.hf_config = self.get_lora_config()
+        self.added_tokens = self.get_added_tokens()
         self.target_modules = self.hf_config["target_modules"]
 
-        # TODO: Support more modules
-        if any(module in self.target_modules for module in ["embed_tokens", "lm_head"]):
+        # TODO: Support lm_head modules
+        if any(module in self.target_modules for module in ["lm_head"]):
             raise ValueError("Not supported yet")
+        # TODO: remove below after we support lm_head
+        # Uncomment below for testing adapter: yard1/llama-2-7b-sql-lora-test (since it has lm_head which is not supported yet)
+        # if "lm_head" in self.target_modules:
```

Comment:
Did I understand correctly that there is no adapter in hugging face that supports only embed but not lm_head?